src/Transition/CreateNewIDFUsingRulesV9_3_0.f90

```diff
@@ -0,0 +1,1387 @@
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 17) THEN
+                   CALL FixFuelTypes(OutArgs(17), NoDiff)
+                 END IF
+
+             CASE('COIL:HEATING:FUEL')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 3) THEN
+                   CALL FixFuelTypes(OutArgs(3), NoDiff)
+                 END IF
+
+             CASE('COIL:HEATING:DX:MULTISPEED')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 16) THEN
+                   CALL FixFuelTypes(OutArgs(16), NoDiff)
+                 END IF
+
+              ! If your original object starts with D, insert the rules here
+
+              ! If your original object starts with E, insert the rules here
+
+             CASE('ENERGYMANAGEMENTSYSTEM:METEREDOUTPUTVARIABLE')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 5) THEN
+                   CALL FixFuelTypes(OutArgs(5), NoDiff)
+                 END IF
+
+             CASE('EXTERIOR:FUELEQUIPMENT')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 2) THEN
+                   CALL FixFuelTypes(OutArgs(2), NoDiff)
+                 END IF
+
+              ! If your original object starts with F, insert the rules here
+
+             CASE('FUELFACTORS')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 ! FixFuelTypes will set NoDiff = .false. if it makes a change
+                 OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                 IF (CurArgs .GE. 1) THEN
+                   CALL FixFuelTypes(OutArgs(1), NoDiff)
+                   ! For fuelfactors, the current IDD choice is Propane, so override that here until #5941 is resolved (standardize fuel types)
```

Comment:
This will be resolved after the remaining PRs on this issue go in?

Reply:
@mitchute Actually, these two lines are old. FixFuelTypes is already setting all propane variants to Propane. I'll delete them to avoid confusion.